extensions/cms/package.json

```diff
@@ -143,6 +143,20 @@
           "isRequired": true,
           "isArray": false
         },
+        {
+          "specialValueType": null,
```

Comment:
are there any side effects from changing this value to null?

Reply:
The only role of appname special type was in hiding it from Advanced Options and setting internal value, so I don't see any side effects of removing that for MSSQL providers.